api/capi/include/nnstreamer-single.h

```diff
@@ -106,6 +141,23 @@ int ml_single_close (ml_single_h single);
  */
 int ml_single_invoke (ml_single_h single, const ml_tensors_data_h input, ml_tensors_data_h *output);
 
+/**
+ * @brief Invokes the model with the given input data and fills the @a output data handle.
+ * @details This function does not allocate new data handle, unlike ml_single_invoke() which returns newly allocated output handle.
+ * @since_tizen 6.5
+ * @param[in] single The model handle to be inferred.
+ * @param[in] input The input data to be inferred.
+ * @param[in] output The output data to be filled.
```

Comment:
Need more info for developers:
```
The output data to be filled.
```
==>
```
The output data to be filled by the API. Output should be preallocated before calling the API.
```

Reply:
Addon : `The memory of each output must be contiguous. However, it is not required for multiple outputs to have contiguous shared memory.`?